src/index.html

```diff
@@ -39,6 +39,7 @@
         audience: '<%= audience %>',
         cpAudience: '<%= cpAudience %>',
         passageEndpoint: '<%= passageEndpoint %>',
+        enableRealUserMonitoring: '<%= enableRealUserMonitoring %>',
```

Comment:
@giantswarm/chapter-front-end Is it necessary to update the webpack config with `enableRealUserMonitoring` ? If yes, how? Currently the build fails with `ERROR in Template execution failed: ReferenceError: enableRealUserMonitoring is not defined`

Reply:
Seems solved.